configurations/performance/latency-decorator-error-thresholds-steps-ent-vnodes.yaml

```diff
@@ -0,0 +1,107 @@
+latency_decorator_error_thresholds:
+  write:
+    "200000":
+      P90 write:
+        fixed_limit: 5
+      P99 write:
+        fixed_limit: 10
+    "300000":
+      P90 write:
+        fixed_limit: 1000
+      P99 write:
+        fixed_limit: 1000
```

Comment:
What units are these numbers?
What is 1000? ms?

Reply:
yes

Reply:
Isn't it way too high?

Reply:
Let's sit together and define the numbers we want to define here.